node/parachain/src/command.rs

```diff
@@ -138,7 +138,7 @@ fn extract_genesis_wasm(chain_spec: &Box<dyn sc_service::ChainSpec>) -> Result<V
 /// Parse command line arguments into service configuration.
 pub fn run() -> Result<()> {
 	let cli = Cli::from_args();
-
+	let account: H160 = cli.run.account_id.unwrap_or_default();
```

Comment:
So if account_id isn't set, it uses H160::default(). Should return an error if None instead of this. (same in node/standalone)

Reply:
done, will push soon